### PR TITLE
Add Robb576

### PR DESCRIPTION
**Pull request by Spandamn**

### CODE REVIEW

**Comment by Spandamn** [comment]
Okay wtf

File: data/mods/ssb/abilities.ts [hunk 1]
```diff
@@ -149,6 +149,69 @@ export const BattleAbilities: {[k: string]: ModdedAbilityData} = {
 			}
 		},
 	},
+	// Robb576
+	"thenumbersgame": {
```
Comment (KrisXV) [inline data/mods/ssb/abilities.ts]:
```suggestion

	// Robb576
	thenumbersgame: {
```

File: data/mods/ssb/abilities.ts [hunk 2]
```diff
@@ -149,6 +149,69 @@ export const BattleAbilities: {[k: string]: ModdedAbilityData} = {
...
+		desc: "Changes the pokemon's form upon switch-in depending on the amount of pokemon still alive on the user's team; Necrozma-Dusk-Mane if 3 or fewer, Necrozma-Ultra if it is the last Pokemon left on the team.",
+		shortDesc: "Changes the pokemon's form upon switch-in depending on the amount of pokemon still alive on the user's team.",
+		name: "The Numbers Game",
+		onStart(pokemon) {
+			if (pokemon.side.pokemonLeft > 3) return;
+			const assignNewMoves = function (poke: Pokemon, moves: string[]) {
```
Comment (KrisXV) [inline data/mods/ssb/abilities.ts]:
```suggestion
			const assignNewMoves = (poke: Pokemon, moves: string[]) => {
```

File: data/mods/ssb/moves.ts [hunk 3]
```diff
@@ -440,6 +440,96 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		target: "normal",
 		type: "Psychic",
 	},
+
+	// Robb576
+	"integeroverflow": {
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
	integeroverflow: {
```

File: data/mods/ssb/moves.ts [hunk 4]
```diff
@@ -440,6 +440,96 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+			if (pokemon.getStat('atk', false, true) > pokemon.getStat('spa', false, true)) move.category = 'Physical';
+		},
+		ignoreAbility: true,
+		isZ: "modium6z",
+		secondary: null,
+		target: "normal",
+		type: "Psychic",
+	},
+
+	"mode5offensive": {
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
	mode5offensive: {
```

File: data/mods/ssb/moves.ts [hunk 5]
```diff
@@ -440,6 +440,96 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+			boosts: {
+				spd: -1,
+			},
+		},
+		multihit: 3,
+		target: "normal",
+		type: "Fighting",
+	},
+
+	"mode7defensive": {
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
	mode7defensive: {
```

File: data/mods/ssb/moves.ts [hunk 6]
```diff
@@ -440,6 +440,96 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+		onTryMovePriority: 100,
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Heal Bell', source);
+			this.add('-anim', source, 'Roar', source);
+		},
+		onHit(pokemon, source) {
+			this.add('-activate', source, 'move: Heal Bell');
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
			this.add('-activate', source, 'move: Mode [7: Defensive]');
```

**Review by KrisXV — CHANGES_REQUESTED** [review]

**Comment by Spandamn** [comment]
@Instructing @HoeenCoder quick quick merge it before more conflicts arise! lol

**Comment by Spandamn** [comment]
i should prolly squash

**Pull request merged**